postgres_ffi/src/xlog_utils.rs

```diff
@@ -264,3 +265,32 @@ pub fn main() {
         tli
     );
 }
+
+//
+// Xlog record parsing routines
+// TODO move here other related code from waldecoder.rs
+//
+#[repr(C)]
```

Comment:
You don't need `repr(C)` if you won't pass this data structure between Rust and C code.

Reply:
I was thinking about the size_of() method for such structs.
Now we define structure sizes as separate constants, i.e SIZEOF_XLOGRECORD.
With repr(C) these will be equivalent and less error-prone.